File: app/src/main/java/com/semisonfire/cloudgallery/ui/main/dialogs/base/DialogListener.java
package com.semisonfire.cloudgallery.ui.main.dialogs.base;

import android.content.DialogInterface;
import android.view.View;

public interface DialogListener {

    void onPositiveClick(DialogInterface dialogInterface);

    void onNegativeClick(DialogInterface dialogInterface);

    void onItemClick(DialogInterface dialogInterface, View view);

}

File: app/src/main/java/com/semisonfire/cloudgallery/ui/dialogs/base/DialogListener.kt
package com.semisonfire.cloudgallery.ui.dialogs.base

import android.content.DialogInterface
import android.view.View

interface DialogListener {

  fun onPositiveClick(dialogInterface: DialogInterface)

  fun onNegativeClick(dialogInterface: DialogInterface)
}

interface BottomDialogListener {
  fun onItemClick(dialogInterface: DialogInterface, view: View)
}
